cmd/snap/cmd_prepare_image.go

```diff
@@ -120,13 +123,15 @@ func (x *cmdPrepareImage) Execute(args []string) error {
 	// level for "snap" command, for seed/seedwriter used by image however
 	// we want real validation.
 	snap.SanitizePlugsSlots = builtin.SanitizePlugsSlots
-
+	imageCustomizations := *new(image.Customizations)
```

Comment:
this can written as imageCustomizations := image.Customizations{}

Reply:
fixed